gdx/test/com/badlogic/gdx/utils/AlignTest.java

```diff
@@ -30,38 +30,71 @@
 public class AlignTest {
 
     @Test
-    public void testIsLeft() {
+    public void testIsLeft () {
         assertTrue(Align.isLeft(Align.left));
         assertFalse(Align.isLeft(Align.right));
+        assertTrue(Align.isLeft(Align.topLeft));
+        assertFalse(Align.isLeft(Align.topRight));
     }
 
     @Test
-    public void testIsRight() {
+    public void testIsRight () {
         assertTrue(Align.isRight(Align.right));
         assertFalse(Align.isRight(Align.left));
+        assertTrue(Align.isRight(Align.topRight));
+        assertFalse(Align.isRight(Align.topLeft));
     }
 
     @Test
-    public void testIsTop() {
+    public void testIsTop () {
         assertTrue(Align.isTop(Align.top));
         assertFalse(Align.isTop(Align.bottom));
+        assertTrue(Align.isTop(Align.topLeft));
+        assertFalse(Align.isTop(Align.bottomLeft));
+        assertTrue(Align.isTop(Align.topRight));
+        assertFalse(Align.isTop(Align.bottomRight));
     }
 
     @Test
-    public void testIsBottom() {
+    public void testIsBottom () {
         assertTrue(Align.isBottom(Align.bottom));
         assertFalse(Align.isBottom(Align.top));
+        assertTrue(Align.isBottom(Align.bottomLeft));
+        assertFalse(Align.isBottom(Align.topLeft));
+        assertTrue(Align.isBottom(Align.bottomRight));
+        assertFalse(Align.isBottom(Align.topRight));
     }
 
-    @Test
-    public void testIsCenterVertical() {
+    @Test public void testIsCenterVertical() {
         assertTrue(Align.isCenterVertical(Align.center));
         assertFalse(Align.isCenterVertical(Align.top));
+        assertFalse(Align.isCenterVertical(Align.bottom));
+        assertFalse(Align.isCenterVertical(Align.topLeft));
+        assertFalse(Align.isCenterVertical(Align.topRight));
+        assertFalse(Align.isCenterVertical(Align.bottomLeft));
+        assertFalse(Align.isCenterVertical(Align.bottomRight));
     }
 
-    @Test
-    public void testIsCenterHorizontal() {
+    @Test public void testIsCenterHorizontal() {
         assertTrue(Align.isCenterHorizontal(Align.center));
         assertFalse(Align.isCenterHorizontal(Align.left));
+        assertFalse(Align.isCenterHorizontal(Align.right));
+        assertFalse(Align.isCenterHorizontal(Align.topLeft));
+        assertFalse(Align.isCenterHorizontal(Align.topRight));
+        assertFalse(Align.isCenterHorizontal(Align.bottomLeft));
+        assertFalse(Align.isCenterHorizontal(Align.bottomRight));
+    }
+
+    @Test
+    public void testCombinations () {
+        assertTrue(Align.isLeft(Align.bottomLeft));
+        assertTrue(Align.isRight(Align.bottomRight));
+        assertTrue(Align.isTop(Align.topRight));
```

Comment:
This test has an awful lot of overlap with the prior tests. `assertTrue(Align.isRight(Align.topRight))` etc. are duplicate lines.

Reply:
I will delete the duplicate lines ! Thank you